features/config/dao/impl/src/main/java/org/opennms/features/config/dao/impl/util/ValidateUsingConverter.java

```diff
@@ -180,23 +152,12 @@ public String getRawSchema() {
     }
 
     @Override
-    public String jaxbObjectToJson(final CONFIG_CLASS configObject) {
-        return xmlMapper.jaxbObjectToJson(configObject);
-    }
-
-    @Override
-    public String jaxbObjectToXml(CONFIG_CLASS configObject) {
-        return xmlMapper.jaxbObjectToXml(configObject);
-    }
-
-    @Override
-    public boolean validate(CONFIG_CLASS configObject) throws RuntimeException {
-        return xmlMapper.validate(configObject);
-    }
-
-    @Override
-    public CONFIG_CLASS xmlToJaxbObject(final String xmlStr) {
```

Comment:
Agree, it would be convenient.
But In general I would like to get away from tying our interface to XML specific functions. Later we will have to deal with other schema formats, I think...
Maybe we add it to the class but not it's interface?